assets/queries/dockerfile/copy_instead_of_add/query.rego

```diff
@@ -0,0 +1,23 @@
+package Cx
+
+CxPolicy [ result ] {
+  resource := input.document[i].command[j]
+  check(resource) 
+  not tarfileChecker(resource.Value, "tar")
+  
+	result := {
+                "documentId": 		input.document[i].id,
+                "searchKey": 	    sprintf("ADD=%s", [resource.Value[0]]),
+                "issueType":		"IncorrectValue", 
+                "keyExpectedValue": "Using Command COPY",
+                "keyActualValue": 	"Using Command ADD"
```

Comment:
Try to identify if possible the command being used inappropriately, so the user can track it down quicker.